src/PasswordHasher/PasswordHasherFactory.php

```diff
@@ -49,7 +49,7 @@ public static function build($passwordHasher)
 
         $hasher = new $className($config);
         if (!($hasher instanceof PasswordHasherInterface)) {
-            throw new RuntimeException('Password hasher must extend AbstractPasswordHasher class.');
+            throw new RuntimeException('Password hasher must implement PasswordHasherInterface.');
```

Comment:
@burzum Test has been added.

I'm wondering if we should allow `App\Authentication\PasswordHasher` namespace for password hashers? Currently only `App\PasswordHasher` is discovered. In fact the same applies for other components: `Authenticators` and `Identifiers`. I think it would be much cleaner if all these authentication related classes could reside under `App\Authentication` namespace.